### DISCUSSION TITLE
No DFU mode

### DISCUSSION THREAD

**Discussion by gitsr-sys** [bitcraze/discussions#1583] [Q&A]
@ataffanel 
I cannot get DFU mode.
Please watch the video.
https://youtube.com/shorts/vEtI1wCHLRA?feature=shared

I followed https://forum.bitcraze.io/viewtopic.php?t=2806
But nothing worked for me.. Please help me to get it factory reset..

**Comment by gitsr-sys** [comment]
@tobbeanton Hello, can you help me to resolve this issue?

**Comment by ataffanel** [comment]
Can you try to click the button one more time on the button to get the led to blink fast, this should enable the DFU mode.
